### PR TITLE
Updating for changes in fontParts and RF3.1

### PR DESCRIPTION
**Pull request by benkiel**
We live in a layers not a font world in UFO3.

### CODE REVIEW

**Comment by miguelsousa** [comment]
@benkiel do you know if these changes break RF1.8 and UFO2?

**Comment by benkiel** [comment]
I don’t, will test

**Comment by benkiel** [comment]
Doesn't, fixed now with a check for RF3

File: AdjustAnchors.roboFontExt/lib/AdjustAnchors.py [hunk 1]
```diff
@@ -36,6 +36,12 @@ def __init__(self):
         self.font = CurrentFont()
         self.glyph = CurrentGlyph()
         self.upm = self.font.info.unitsPerEm
+        if int(roboFontVersion.split(".")[0]) >= 3:
```
Comment (miguelsousa) [inline AdjustAnchors.roboFontExt/lib/AdjustAnchors.py]: What about 
```python
self.rf3 = int(roboFontVersion.split(".")[0]) >= 3
```
Will that work?

**Comment by benkiel** [comment]
Yes, will change

**Comment by benkiel** [comment]
Updated.

**Comment by miguelsousa** [comment]
thanks Ben. I'll bump the version and merge shortly

**Comment by miguelsousa** [comment]
Merged via f5dfcda290e0c35511ab15e1fe725df224a49b57

**Pull request closed without merging**